webrtc.html

```diff
@@ -2089,6 +2136,52 @@
           </ul>
         </div>
       </section>
+
+      <section>
+        <h4>RTCUnassignedMediaEvent</h4>
+
+        <p>
+          The <code><a>unassignedmedia</a></code> event
+          on <code><a>RTCPeerConnection</a></code> uses
+          the <code><a>RTCUnassignedMediaEvent</a></code> interface.
+        </p>
+
+        <p>
+          <dfn data-lt="Fire an unassigned media event">Firing an
+            <code><a>RTCUnassignedMediaEvent</a></code> event named
+            <var>e</var></dfn> means that an event with the name <var>e</var>,
+          which does not bubble (except where otherwise stated) and is not
+          cancelable (except where otherwise stated), and which uses the
+          <code><a>RTCUnassignedMediaEvent</a></code> interface MUST be created and
+          dispatched at the given target.
+        </p>
+
+        <dl class="idl" data-merge="RTCUnassignedMediaEventInit"
+            title="interface RTCUnassignedMediaEvent : Event">
+          <dt>Constructor(DOMString type, RTCUnassignedMediaEventInit
+          eventInitDict)</dt>
+
+          <dt>readonly attribute unsigned long ssrc</dt>
+          <dd>The Synchronization source (SSRC) number of the packet.</dd>
+
+          <dt>readonly attribute unsigned byte payloadType</dt>
+          <dd>The RTP payload type of the packet.</dd>
+
+          <dt>readonly attribute DOMString? mid</dt>
+         <dd>The value of any mid header extension on the discarded media
+          packet, if that header extension is present.</dd>
+        </dl>
+
+        <dl class="idl" title="dictionary RTCUnassignedMediaEventInit : EventInit">
+          <dt>unsigned long ssrc</dt>
+          <dd>See <a><code>RTCUnassignedMediaEvent.ssrc</code></a></dd>
+          <dt>unsigned byte payloadType</dt>
+          <dd>See <a><code>RTCUnassignedMediaEvent.payloadType</code></a></dd>
```

Comment:
Wasn't this syntax (type.field) what was objected to because it looks like an accessor to a static? @adam-be has been fixing some of these - he may have comments.